src/platform/mhv/util/helpers.js

```diff
@@ -206,7 +207,7 @@ export const makePdf = async (
   } catch (error) {
     // Reset the pdfModulePromise so subsequent calls can try again
     pdfModulePromise = null;
-
+    datadogRum.addError(sentryErrorLabel);
```

Comment:
I'm not sure what the `sentryErrorLabel` value would be here. Let's use the error "PDF generation failed"

Reply:
I looked at those messages and I think that's actually what we want. Disregard my comment.